### ISSUE TITLE
Can't check for updates

### ISSUE THREAD

**Issue by junocomp** [armandg/sol3rd#6]
```
$ sudo sol3rd -u
Traceback (most recent call last):
  File "/usr/bin/sol3rd", line 11, in <module>
    load_entry_point('sol3rd==0.3.1', 'console_scripts', 'sol3rd')()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 572, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2755, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2408, in load
    return self.resolve()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2414, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python2.7/site-packages/sol3rd/sol3rd.py", line 12, in <module>
    from sol3rd.__version__ import version
ImportError: No module named __version__

```

**Comment by geirawsm** [comment]
This script has python 3.x+ version as a requirement. Try that instead and please report back if it doesn't help.

**Comment by junocomp** [comment]
I have python3 installed but is not working, unless I am doing something different. Can you show me how to do it?

**Comment by geirawsm** [comment]
Did you install the package like this: `pip3 install sol3rd`?

This makes sure that it knows to use python3 instead of python2.

**Comment by junocomp** [comment]
You're a genius (and I'm not). It works now. Thanks!

**Comment by geirawsm** [comment]
Wow, didn't expect that, thanks :D

**Comment by junocomp** [comment]
Maybe you should update the ReadME

**Comment by geirawsm** [comment]
Oh I will, thanks for pointing it out.

**Comment by geirawsm** [comment]
Fixed in da01e2d.
